test/JMeter/README.md

```diff
@@ -0,0 +1,15 @@
+**Stress Test Guide:**
+
+Stress testing on the ingress was performed using [JMeter](https://jmeter.apache.org/download_jmeter.cgi).
+Initial testing was performed in GUI mode, however if pushing the limits it can be run from CLI as well.
+
+In order to stress test open the `PerformanceTest.jmx` file with JMeter and edit these properties:
+- In HTTP Request Defaults, edit `Server Name or IP` field to point to your helloworld application.
```

Comment:
I suggest including UI screen shots with annotation. And also including CLI commands as alternative. Ideally CLI but if GUI is more intuitive then include both please.